lte/gateway/c/core/oai/tasks/mme_app/mme_app_test_serialization.cpp

```diff
@@ -0,0 +1,297 @@
+/**
```

Comment:
I would suggest, moving this file along with the `..._generated.h` files from flatbuffer compiler to a folder called `lte/gateway/c/core/oai/experimental` and then modify CMakeLists to include this as a library, adding this to the `tasks` and `include` folders does not seem like the right place, thoughts @ssanadhya ?